### PR TITLE
Added title icon

### PR DESCRIPTION
**Pull request by GeethOnion**
Added title icon

- [x] I've checked that this isn't a duplicate pull request.

Is it regarding a new swag proposal or something else?  
Fill the corresponding section below:

(New Swag Proposal)  
Q. Have you received swag from this?  
Q. How long did it take?  
Q. Does it ship worldwide?  

(Something else)  
Please describe your changes here:

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,10 +1,12 @@
...
-<body>
-   <p>Redirecting to: 
-   <a href="http://devswag.io">devSwag</a></p>
-</body>
-</html>
+<!DOCTYPE html>
+<html>
+<head>
+    <title>devSwag</title>
+    <link rel="icon" type="image/ico" href="C:\Users\GeethmakaDissanayake\Desktop\New folder (2)\swag-for-dev\site\src\img\logo.png" />
```
Comment (swapagarwal) [inline index.html]: This should be a relative path, something like `site/src/img/logo.png`.

**Comment by swapagarwal** [comment]
Please rebase this on master.

**Comment by swapagarwal** [comment]
Oops, sorry. I didn't mean to close this. 😇
Can you reopen it?

File: index.html (hunk 1 above)
Comment (GeethOnion) [reply index.html]: Oh sorry totally it's my fault

**Pull request closed without merging**